### PR TITLE
[plugin.video.tele5_de] 1.1.7

### PR DESCRIPTION
**Pull request by tyl0re**
### Description
Plugin for the Videos of the Television Station Tele5 from Germany. An ealrier Version has exists from Addonscriptor, wo ist active anymore

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.tele5_de/default.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+import HTMLParser
+import json
+
+reload(sys)
+sys.setdefaultencoding('utf-8')
+
+#addon = xbmcaddon.Addon()
+#addonID = addon.getAddonInfo('id')
+addonID = 'plugin.video.tele5_de'
+addon = xbmcaddon.Addon(id=addonID)
```
Comment (enen92) [inline plugin.video.tele5_de/default.py]: no need for id here

File: plugin.video.tele5_de/default.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+socket.setdefaulttimeout(30)
+pluginhandle = int(sys.argv[1])
+xbmcplugin.setContent(pluginhandle, 'movies')
+translation = addon.getLocalizedString
+xbox = xbmc.getCondVisibility("System.Platform.xbox")
```
Comment (enen92) [inline plugin.video.tele5_de/default.py]: This does not exist anymore

File: plugin.video.tele5_de/default.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+            paramSplits = paramsPair.split('=')
+            if (len(paramSplits)) == 2:
+                paramDict[paramSplits[0]] = paramSplits[1]
+    return paramDict
+
+
+def addLink(name, url, mode, iconimage, desc="", duration="",count=0,dadd=0,genre="",episode=0,season=0):
+    u = sys.argv[0]+"?url="+urllib.quote_plus(url)+"&mode="+str(mode)+"&count="+str(count)
+    ok = True
+    liz = xbmcgui.ListItem(name, iconImage="DefaultVideo.png", thumbnailImage=iconimage)
```
Comment (enen92) [inline plugin.video.tele5_de/default.py]: please use setArt

File: plugin.video.tele5_de/default.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    liz.setProperty('IsPlayable', 'true')
+    liz.addContextMenuItems([(translation(30004), 'RunPlugin(plugin://'+addonID+'/?mode=queueVideo&url='+urllib.quote_plus(u)+'&name='+str(name)+'&thumb='+urllib.quote_plus(iconimage)+')',)])
+    ok = xbmcplugin.addDirectoryItem(handle=int(sys.argv[1]), url=u, listitem=liz)
+    return ok
+
+
+def addDir(name, url, mode, iconimage, desc="", duration=""):
+    u = sys.argv[0]+"?url="+urllib.quote_plus(url)+"&mode="+str(mode)
+    ok = True
+    liz = xbmcgui.ListItem(name, iconImage="DefaultFolder.png", thumbnailImage=iconimage)
```
Comment (enen92) [inline plugin.video.tele5_de/default.py]: same here

File: plugin.video.tele5_de/resources/language/English/strings.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (enen92) [inline plugin.video.tele5_de/resources/language/English/strings.xml]: Please use strings.po format instead of xml

Comment (tyl0re) [inline plugin.video.tele5_de/resources/language/English/strings.xml]: All should be Corrected

**Comment by tyl0re** [comment]
All should be Corrected

**Pull request merged**